docker-compose.yml

```diff
@@ -334,6 +336,104 @@ services:
       volumes:
         - ./apps/collections/log:/app/log
 
+  publisher: &publisher
+      build: apps/publisher
+      depends_on:
+        - publishing-api
+        - rummager
```

Comment:
Is Rummager definitely required as a dependency? I thought it would rely on the Publishing API to communicate with Rummager.

Reply:
Good catch!